Homework/src/main/java/io/codelex/testtwo/exercise2/Basket.java

```diff
@@ -0,0 +1,33 @@
+package io.codelex.testtwo.exercise2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket<T> {
+    private List<T> list;
+    private int elementCount=0;
+    private T object;
```

Comment:
Šo mainīgo laikam definēji, lai varētu pārbaudīt, ka nāmā lieta, ko pievieno grozam ir tāda pat (pēc tipa).
Šis nebūs īsti pareizi, jo to varēja attrisināt daudz vienkāršāk: jau pašā klasē ir definēts tips, kādu tā saturēs T, bet apakšā metode saņem tipu E, ja viņa saņemtu T, tad jau kompilators pārbauda vai ir padots pareizais tips. Attiecīgi tad pazūd jēga šim mainīgajam, kā arī nav jāmet kļūda pat input mismatch, jo nebūs vispār iespējams ielikt Basket<Apple> objektu ar tipu Mushroom.

Reply:
Man bija jau saprašana par to ka nebūs iespējams iebāzt divu veidu objektus, tikai ši daļa man mazliet sajauca prātu
#### NB! Basket can store only one kind of items - only apples, only mushrooms etc, it cannot store both apples and mushrooms at the same time.

Create a test to test functionality and check that its possible to create basket with apples, basket with mushrooms and that exceptions are thrown.